libraries/AP_Common/missing/cmath

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include_next <cmath>
+
+#if defined(HAVE_CMATH_ISFINITE) && !defined(HAVE_MATH_ISFINITE)
```

Comment:
You're checking whether they are defined or not to know if the feature is respectively supported or not.  I totally understand why you're doing this: because that's the way waf creates the configuration header. However, we already have other `HAVE_*` style macros that are checked differently, for example:

```c++
#if defined(HAVE_STD_NULLPTR_T) && !HAVE_STD_NULLPTR_T
```

We need consistency. I don't see a problem (besides verbosity) with using the same approach used with `HAVE_STD_NULLPTR_T`, since that would work fine with the way Waf writes the configuration header.

Maybe @lucasdemarchi wants to give an opinion here?